client/my-sites/stats/index.js

```diff
@@ -206,6 +208,18 @@ export default function() {
 			page( '/stats/reset-first-view', statsController.resetFirstView, makeLayout, clientRender );
 		}
 
+		page(
+			`/stats/wordads/:period(${ validPeriods.join( '|' ) })/:site`,
+			siteSelection,
+			navigation,
+			statsController.wordAds,
+			makeLayout,
+			clientRender
+		);
+
+		// Anything else should redirect to default WordAds stats page
+		page( '/stats/wordads(.*)', statsController.redirectToDefaultWordAdsPeriod );
```

Comment:
`'/stats/wordads/*'` is slightly better. We don't really want to match paths like `/stats/wordadsense`.